.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+indent_size = 2
```

Comment:
Hehe, at the risk of taking this off topic twice on one thread we did actually have a Numix license for our closed source projects. It should in theory be somewhere in this repos git history because it was the license for the private Square repo before it was merged into here.